src/wp-includes/blocks.php

```diff
@@ -716,6 +716,16 @@ function excerpt_remove_blocks( $content ) {
 		'core/group',
 	);
 
+	/**
+	 * Filters the list of blocks that can be used as wrapper blocks,
+	 * allowing excerpts to be generated from their `innerBlocks`<div class=""></div>
```

Comment:
Is the empty `<div>` here intentional?

Reply:
:grimacing: Nope, apparently VSCode autocompletes `.` to a div via Emmet :facepalm: 
Removed, thanks for catching that!

Reply:
No problem! Just wanted to make sure I wasn't missing anything.